### PR TITLE
More documentation for Cache. Updated FAQ to include database Locked error 

### PR DESCRIPTION
**Pull request by balakreshnan**
Database Locked error in Azure Machine Learning compute instance

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by ekzhu** [comment]
raise awareness to @sonichi regarding cache that relies on local filesystem access

**Review by sonichi — COMMENTED** [review]
@ekzhu @vijaykramesh Do we need to update this part after #1222 is merged?

**Comment by ekzhu** [comment]
The doc needs to be update again once #1222  is merged. Let's wait for that one to merge first.

File: website/docs/FAQ.md [hunk 1]
```diff
@@ -192,6 +192,21 @@ Please refer to https://microsoft.github.io/autogen/docs/reference/agentchat/con
 The "use_docker" arg in an agent's code_execution_config will be set to the name of the image containing the change after execution, when the conversation finishes.
 You can save that image name. For a new conversation, you can set "use_docker" to the saved name of the image to start execution there.
 
+## Database Locked error
+
+When using VMs such as Azure Machine Learning compute instances, you may encounter a "database locked error".
+There are two workarounds.
+1. Set `cache_root_path` to a location where the code has access. For example, `OpenAIWrapper.cache_root_path=/tmp/.cache`.
```
Comment (sonichi) [inline website/docs/FAQ.md:199]: This is outdated after #1222

**Comment by ekzhu** [comment]
I added more documentation for `Cache`, added `Cache` to top-level module so we can do `from autogen import Cache`

**Review by sonichi — APPROVED** [review]

**Pull request merged**